kinde_sdk/frontend/api/roles_api.py

```
@@ -0,0 +1,326 @@
+    Contact: support@kinde.com
+    Generated by OpenAPI Generator (https://openapi-generator.tech)
+
+    Do not edit the class manually.
+"""  # noqa: E501
+
+import warnings
+from pydantic import validate_call, Field, StrictFloat, StrictStr, StrictInt
+from typing import Any, Dict, List, Optional, Tuple, Union
+from typing_extensions import Annotated
+
+from pydantic import Field, StrictInt, StrictStr
```

Ruff (0.12.2)

Redefinition of unused `Field` from line 16

Remove definition: `Field`

(F811)